ansible/playbooks/jenkins/worker/create.yml

```diff
@@ -102,8 +102,9 @@
 - hosts:
   - test
   - release
-  - "!test-joyent-freebsd10-x64-2"
-  - "!test-rackspace-freebsd10-x64-1"
+  - "!test-packetnet-ubuntu1604-x64-1"
+  - "!test-packetnet-ubuntu1604-x64-2"
+  - "!test-softlayer-ubuntu1604-x64-1"
```

Comment:
There is a "Set up linter servers" section just above this, that runs exclusively on the two hosts being removed here. Shouldn't that section be completely removed? There are some bits about core-validate-commit there that I don't know if are still relevant in some scenario.

Reply:
Yeah, missed that, also missed core-validate-commit. There is a job: https://ci.nodejs.org/job/node-test-commitmsg/ that runs it but it hasn't had any runs since the last build cleanup (~1 week). Does that mean it's disabled? Maybe this is all handled happily in Travis now and in node-core-utils. @nodejs/tooling can you weigh in here? Can we remove node-test-commitmsg from Jenkins and the associated server resources for running it?

Reply:
We are definitely running core-validate-commit as part of node-core-utils and on Travis for PRs and that should be sufficient for the normal PR workflows for core.

An old issue, https://github.com/nodejs/build/issues/793, proposed adding https://ci.nodejs.org/job/node-test-commitmsg/ to `node-test-pull-request` but that didn't happen.

Reply:
Integrating core-validate-commit into my .ci.yml proposal; a failing v8.x: https://ci.nodejs.org/job/rv-test-commit-linux-docker/129/label=docker-host-x64,linux_x64_container_suite=lint-commit/console, ci.yml entry (see 'tests->lint-commit' block): https://github.com/rvagg/io.js/blob/rvagg/testing-containers-v8.x/.ci.yml

I might archive that Jenkins job since it's not in use, it can be unarchived if there's reason to before it eventually gets deleted.